.github/workflows/auto-merge-on-demand.yml

```diff
@@ -0,0 +1,95 @@
+name: Auto Merge On Demand
+on:
+  schedule:
+    # Workflow runs every 45 minutes
+    - cron: '*/45 * * * *'
+  workflow_dispatch:
+    inputs:
+      pr-number:
+        description: 'Pull Request number/s ; when not provided, the workflow will run for all open PRs'
+        required: true
+        default: '0'
+
+permissions:
+  contents: read
+
+jobs:
+  # Get all open PRs
+  gather-pull-requests:
+    if: github.repository == 'sclorg/nginx-container'
```

Comment:
I would prefer to have something like this https://github.com/sclorg/nginx-container/blob/master/.github/workflows/build-and-push.yml#L10